code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -273,4 +274,8 @@ Pipelines + Other Objects -> Pipe network
 
 
 /obj/machinery/atmospherics/proc/can_crawl_through()
-	return 1
\ No newline at end of file
+	return 1
+
+/obj/machinery/atmospherics/proc/returnPipenets()
+	return

```

Comment:
I just checked and list(1,2,3) ^ null = list(1,2,3,null), null ^ list(1,2,3) = list(1,2,3). I'll change it to be more explicit anyway.